src/LoginFormContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  requestLogin,
+  changeLoginFields,
+  logout,
+} from './actions';
+
+import { get } from './utils';
+
+import LoginForm from './LoginForm';
+import LogoutForm from './LogoutForm';
```

Comment:
```suggestion
import LoginForm from './LoginForm';
import LogoutForm from './LogoutForm';

import {
  requestLogin,
  changeLoginFields,
  logout,
} from './actions';

import { get } from './utils';
```
저희는 

리액트 관련된 건 위로 몰아 놓고, 컴포넌트도 리액트니까요.

util은 보통 마지막에 둡니다.ㅎㅎ

Reply:
리액트에 관련된것을 상위로! 명심하겠습니다 ㅎㅎ